scripts/build/fiwarelab/build-fiwarelab-image.sh

```diff
@@ -89,6 +89,10 @@ outgoing HTTP calls from the Context Broker. The rush instance is installed in /
 An instance of Redis compatible with Rush is also installed. More information about Rush can be
 found in its Github page: https://github.com/telefonicaid/Rush
 
+However, neither Rush nor Redis are started by default so you need to use /etc/init.d/redis
```

Comment:
@aaranz, yo told me about this improvement in welcome message time ago. Just in case you want to have a look how it looks like now...